src/drive/web/modules/views/OnlyOffice/components/FileDeletedModal.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { useClient } from 'cozy-client'
+import { ConfirmDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import Buttons from 'cozy-ui/transpiled/react/Buttons'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Alert from 'cozy-ui/transpiled/react/Alert'
+
+import { DOCTYPE_FILES } from 'drive/lib/doctypes'
+import { useOnlyOfficeContext } from 'drive/web/modules/views/OnlyOffice/OnlyOfficeProvider'
+
+const FileDeletedModal = () => {
+  const { fileId, setFileDeleted } = useOnlyOfficeContext()
+  const navigate = useNavigate()
+  const client = useClient()
+  const { t } = useI18n()
+
+  const [isErrorAlertDisplayed, setErrorAlertDisplayed] = useState(false)
+  const [isBusy, setBusy] = useState(false)
+
+  const restore = async () => {
+    setErrorAlertDisplayed(false)
+    setBusy(isBusy)
+    try {
+      const resp = await client.collection(DOCTYPE_FILES).restore(fileId)
+      navigate(`/onlyoffice/${resp.data.id}`)
+      setFileDeleted(false)
+    } catch {
+      setErrorAlertDisplayed(true)
+    } finally {
+      setBusy(false)
+    }
+  }
+
+  const goBack = () => {
+    // TODO : Go to folder with app slug
+    navigate(-2)
```

Comment:
Please no number ;). Path path path :)

Reply:
I'm reworking so that I don't need to go back to the browser history anymore